recipes/virusrecom/meta.yaml

```
@@ -45,3 +45,5 @@ about:
 extra:
   identifiers:
     - doi:10.1093/bib/bbac513
+  recipe-maintainers:
+    - ZhiJianZhou
```

```suggestion
    - ZhijianZhou01
```
Please use your GitHub username here so you can be tagged by the autobump bot whenever there are new version releases. Otherwise, LGTM!